batch.go

```diff
@@ -62,6 +63,14 @@ func NewBatch(bh *BatchHeader) (Batcher, error) {
 	return nil, &FileError{FieldName: "StandardEntryClassCode", Value: bh.StandardEntryClassCode, Msg: msg}
 }
 
+func (batch *batch) Create() error {
```

Comment:
These are only here so `batch` implements `Batcher`. Should I document that?

Reply:
History is that batch was originally Batch because it is basically the same for PPD/CCD. As we added additional SEC types we made Batch->batch as almost a parent class of each concrete Batch<SEC> type.

Reply:
That's fine. NewBatch and all the methods on `batch` are useful and I'd rather not re-implement (code gen) those across every BatchPPD/BatchWEB struct.

Reply:
This is a good example of when traits, inheritance, or generics would be nice in GoLang.

Reply:
Yea.. Mixing struct's and interface's can get gnarly quick.